doc/libpmemobj/pmemobj_action.3.md

```diff
@@ -61,6 +61,7 @@ PMEMoid pmemobj_reserve(PMEMobjpool *pop, struct pobj_action *act,
 	size_t size, uint64_t type_num);
 PMEMoid pmemobj_xreserve(PMEMobjpool *pop, struct pobj_action *act,
 	size_t size, uint64_t type_num, uint64_t flags);
+void pmemobj_defer_free(PMEMobjpool *pop, PMEMoid oid, struct pobj_action *act);
```

Comment:
?

Reply:
To implement this patch I needed the changes I made in defer free PR. In general, I'd like to split this patch into multiple commits (I had to squash due to very disruptive rebase), but for now, this patch includes #2669